frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -114,7 +114,24 @@
 //! good solution is queued, then the fallback strategy [`pallet::Config::Fallback`] is used to
 //! determine what needs to be done. The on-chain election is slow, and contains no balancing or
 //! reduction post-processing. See [`onchain::OnChainSequentialPhragmen`]. The
-//! [`FallbackStrategy::Nothing`] should probably only be used for testing, and returns an error.
+//! [`FallbackStrategy::Nothing`] just returns an error, and enables the [`Phase::Emergency`].
+//!
+//! ### Emergency Phase
+//!
+//! If, for any of the below reasons:
+//!
+//! 1. No signed and unsigned solution submitted
+//! 2. Internal error
+//! 3. Fallback being `None`
```

Comment:
```suggestion
//! 1. No signed or unsigned solution submitted & Fallback is `None`
//! 2. Internal error
```

Maybe?

Reply:
Also, does internal error refer to just errors during `T::ElectionProvider::elect` or could errors in other logic branch force `Phase::Emergency`?

Reply:
as far as I can see the internal error here is only if fallback strategy exist and is failing.